.env.sample

```diff
@@ -0,0 +1,4 @@
+TEST_PK_1='put your private key here'
+TEST_Pk_2='put your private key here'
+TEST_Pk_3='put your private key here'
```

Comment:
this keys is not need right now. could you please remove it?

Reply:
Sure thing!